fuzz_test.go

```diff
@@ -102,8 +102,9 @@ func TestFuzz_basic(t *testing.T) {
 }
 
 func checkFailed(t *testing.T, failed map[string]int) {
+	t.Helper()
 	for k, v := range failed {
-		if v > 8 {
+		if v > 18 {
```

Comment:
Hm, I don't like the long distance between the magic numbers "20" and "18", however I'm not asking for a change since you didn't cause this problem and it looks like you updated all the right places.